.github/vale-styles/messaging/consistent-terms.yml

```diff
@@ -0,0 +1,22 @@
+# The consistent-terms rule ensures that terms with frequently used variations
+# appear consistently in the docs.
+extends: substitution
+scope:
+  - heading
+  - table
+  - list
+  - paragraph
+message: For consistent product messaging in the docs, use '%s' instead of '%s'.
+level: error
+# Ignoring case because this rule is about word choice, rather than its
+# presentation/format.
+ignorecase: true
+swap:
+  2FA: MFA
+  'second[ -]factor': multi-factor
+  'auth and proxy': Auth Service and Proxy Service
+  'auth server|auth node': \"Auth Service\" (or \"Auth Service instance\" for a specific node)
+  'Auth Services': Auth Service instances
+  'Teleport open source|open source Teleport': Teleport Community Edition
+  'OSS Teleport|Teleport OSS': Teleport Community Edition
+  'Teleport Cloud': \"Teleport Team\" or \"Teleport Enterprise Cloud\"
```

Comment:
Not sure about this one.

"Teleport Cloud" means any of our SaaS offerings.  "Teleport Team" and "Teleport Enterprise Cloud" are different tiers, but they are both Teleport Cloud.

I think we should keep a way to refer to "our SaaS" independent of which tier we're speaking of.

Reply:
That makes sense to me and is consistent with how our docs are now. I'll plan to add "Teleport Cloud" to the Core Concepts page.